precompile/contracts/nativeminter/contract_test.go

```diff
@@ -158,6 +160,58 @@ var tests = map[string]testutils.PrecompileTest{
 		ReadOnly:    false,
 		ExpectedErr: vmerrs.ErrOutOfGas.Error(),
 	},
+	"mint doesn't log if D fork is not active": {
+		Caller:     allowlist.TestEnabledAddr,
+		BeforeHook: allowlist.SetDefaultRoles(Module.Address),
+		ChainConfigFn: func(ctrl *gomock.Controller) precompileconfig.ChainConfig {
+			config := precompileconfig.NewMockChainConfig(ctrl)
+			config.EXPECT().IsDUpgrade(gomock.Any()).Return(false).AnyTimes()
+			return config
+		},
+		InputFn: func(t testing.TB) []byte {
+			input, err := PackMintInput(allowlist.TestEnabledAddr, common.Big1)
+			require.NoError(t, err)
+			return input
+		},
+		SuppliedGas: MintGasCost,
+		ReadOnly:    false,
+		ExpectedRes: []byte{},
+		AfterHook: func(t testing.TB, stateDB contract.StateDB) {
+			// Check no logs are stored in state
+			logsTopics, logsData := stateDB.GetLogData()
+			require.Len(t, logsTopics, 0)
+			require.Len(t, logsData, 0)
+		},
+	},
```

Comment:
Isn't this the same as "mint funds from enabled address" above?

Reply:
indeed, they're just testing different after hooks and have different focus. I'd like to keep them as they're for the clarity.